site/current-book/metron-analytics/metron-profiler-common/index.html

```diff
@@ -289,7 +289,16 @@ <h4><a name="profile"></a><tt>profile</tt></h4>
 <h4><a name="foreach"></a><tt>foreach</tt></h4>
 <p><i>Required</i></p>
 <p>A separate profile is maintained &#x2018;for each&#x2019; of these.  This is effectively the entity that the profile is describing.  The field is expected to contain a Stellar expression whose result is the entity name.</p>
-<p>For example, if <tt>ip_src_addr</tt> then a separate profile would be maintained for each unique IP source address in the data; 10.0.0.1, 10.0.0.2, etc.</p></div>
+<p>For example, if <tt>ip_src_addr</tt> then a separate profile would be maintained for each unique IP source address in the data; 10.0.0.1, 10.0.0.2, etc.</p>
```

Comment:
Hi @DimDroll Thanks for the contribution.  This change actually needs to be made in the file `metron-analytics/metron-profiler-common/README.md`, which is used to generate this HTML page.